azure-mssql-db.yml

```diff
@@ -27,60 +27,29 @@ plans:
   id: fd07d12b-94f8-4f69-bd5b-e2c4e84fafc1
   description: 'SQL Server latest version. Instance properties: General Purpose - Serverless ; 0.5 - 2 cores ; Max Memory: 6gb ; 5 GB storage ; auto-pause enabled after 1 hour of inactivity'
   display_name: "Small"
-  properties:
-    subsume: false
 - name: medium
   id: 3ee14bce-33e8-4d02-9850-023a66bfe120
   description: 'SQL Server latest version. Instance properties: General Purpose - Provisioned ; Provisioned Capacity ; 8 cores ; 200 GB storage'
   display_name: "Medium"
   properties:
     cores: 8
     max_storage_gb: 200
-    subsume: false
```

Comment:
Do we have currently passing tests to check upgrading from a previous version of the broker which still supported subsume property?

I believe we determined that no-one was using subsume, so my concern is whether removing the property can cause any reconciliation issues with information in the internal DB.

If information in the internal DB contains any data associated to subsume property. What would happen when that is verified against this YAML?

Reply:
I guess that is the purpose of the new test:
`update_and_upgrade_mssql_db_subsume_test.go`

Reply:
That's the objective of the test:

The subsume functionality has been removed. We create a test in which we subsume a service and change the plan to remove old references. Then we upgrade and verify we can change the service without problems. This will be the upgrade process for subsumed services. I will create the documentation when finishing the fail-over test.